jsf/injection/src/main/resources/META-INF/services/org.apache.myfaces.spi.InjectionProvider

```diff
@@ -0,0 +1 @@
+org.jboss.as.jsf.injection.MyFacesInjectionProvider
```

Comment:
I think this file can be removed since only the org.apache.myfaces.spi.InjectionProviderFactory services file will actually get used.